launchable/utils/session.py

```diff
@@ -20,7 +19,14 @@ def _session_file_path(build_name: str) -> Path:
 
 
 def _get_session_id():
-    id = os.getsid(os.getpid())
+    if sys.platform == "win32":
```

Comment:
I don't know the concept and the practical use of a logon session well enough to be able to say if this is good or bad. I assume you looked into this somewhat in order to get to this code. Is there any journal around that effort?

I'd think one key test is to see if two different shell invocations in GitHub Action would produce the same id under this scheme. If not, we have a problem.

Reply:
I haven't written any journal about it. I referred [a CPAN module](https://metacpan.org/pod/Win32::Monitoring::Session).

Testing with two different shell invocations is a good idea. I'm going to try it.